godtools/App/Share/Extensions/View+AppLifecycle.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  View+AppLifecycle.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 1/25/24.
+//  Copyright © 2024 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+extension View {
+    
+    // NOTE: SwiftUI supports lifecycle observation through `ScenePhase` (https://developer.apple.com/documentation/swiftui/scenephase),
```

Comment:
Hmm this is interesting, I haven't heard of this before.  Will definitely have to take a look into this.

Reply:
Looks like we'd also want to move away from using UIApplicationDelegate and use the SceneDelegate.  Something I'd like to shift towards soon too. https://www.kairadiagne.com/2019/11/18/adopting-the-new-scene-delegate